src/bulletproof.php

```diff
@@ -114,21 +114,21 @@ public function __construct(array $_files = array())
      * @param mixed $offset
      * @param mixed $value
      */
-    public function offsetSet($offset, $value) {}
+    public function offsetSet($offset, $value) : void {}
 
     /**
      * \ArrayAccess unused method
      * 
      * @param mixed $offset
      */
-    public function offsetExists($offset){}
+    public function offsetExists($offset) : bool {}
```

Comment:
There seems to be not return statement when an bool is expected.

Reply:
That's the correct return type for the method. Are you getting errors?